bin/kafka-server-start.sh

```diff
@@ -19,6 +19,14 @@ then
 	echo "USAGE: $0 [-daemon] server.properties [--override property=value]*"
 	exit 1
 fi
+
+for arg in "$@"; do
+    if [[ "$arg" == *.properties ]]; then
+        [[ "$arg" == *"config/kraft"* ]] && echo "Warning: If using default properties file, config/kraft path has been deprecated"
+        break
+    fi
+done
```

Comment:
Why is this needed to address the intent of the Jira? It is odd for a script to be checking the path of a file. The CLI should only be concern with the content of the file.

Reply:
We are making a breaking change without a deprecation cycle, so this is trying to provide a warning related to that. I haven't checked if the warning is effective though - good to think through that.

Reply:
No, the config files are moving to a different path.